pom.xml

```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.424</version>
+    <version>1.580.1</version>
```

Comment:
I was able to make it work with 1.554. For earlier version running with the newer guava version required for JGiven makes the other tests fail.